wp-job-manager.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: WP Job Manager
  * Plugin URI: https://wpjobmanager.com/
  * Description: Manage job listings from the WordPress admin panel, and allow users to post jobs directly to your site.
- * Version: 1.34.1
+ * Version: 1.34.2
```

Comment:
There's a "Tested up to" in this file as well.

Reply:
https://github.com/Automattic/WP-Job-Manager/pull/2018/commits/55dce87944b37f94da716a9a2b01ae3b73617b86